frontend/src/components/admin/Pagination.tsx

```diff
@@ -1,26 +1,36 @@
 import React from 'react'
+import ReactPaginate from 'react-paginate'
+
+interface Props {
+  length: number
+  pageNumber: number
+  pageCount: number
+  actions: {
+    changePage: any
+  }
+}
+
```

Comment:
```suggestion
type Props = {
  length: number
  pageNumber: number
  pageCount: number
  actions: {
    changePage: any
  }
}

```

i have commented this multiple times. please take note